requirements/test.txt

```diff
@@ -1,5 +1,6 @@
+-e .
```

Comment:
It's better to have installed app and then optional stuff then the other way, I think.